flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -134,7 +134,8 @@ private static Notification createNotification(Context context, NotificationDeta
                 .setContentIntent(pendingIntent)
                 .setPriority(notificationDetails.priority)
                 .setOngoing(BooleanUtils.getValue(notificationDetails.ongoing))
-                .setOnlyAlertOnce(BooleanUtils.getValue(notificationDetails.onlyAlertOnce));
+                .setOnlyAlertOnce(BooleanUtils.getValue(notificationDetails.onlyAlertOnce))
+                .setShowWhen(BooleanUtils.getValue(notificationDetails.setShowWhen));
```

Comment:
I believe this needs amending. Any notification that was scheduled before this update was released would have a null value for the property when loaded from shared preferences